### DISCUSSION TITLE
Wrong time on continuous GoPro lab preset photos

### DISCUSSION THREAD

**Discussion by kelp-videos** [gopro/labs#738] [General]
I am using GoPro lab presets on a GoPro 11 to take one photo every 30 seconds, from 8am-8pm. I just tested out the preset and when I opened the micro SC card's files, all of my photos had the same time stamp (the time the first photo was taken). How do I ensure that the GoPro records the actual time each photo is taken rather than just the time the first preset photo was taken?

**Comment by dnewman-gpsw** [comment]
This is not a Labs bug, likely a user error of setting the time inside a looping script.  Easy mistake.  The problem is the time is "fixed" to the scan time as it is set wrongly on each loop.  So don't do that.  Never set the time inside a loop.

When asking for help of a failing script, please share the QR code scanned.
